docs/perf_counters.md

```diff
@@ -1,5 +1,3 @@
-<a name="perf-counters" />
```

Comment:
can you leave this in please? (or replace it with the equivalent github markdown anchor method)